cli/args/flags.rs

```diff
@@ -2665,7 +2670,15 @@ fn test_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
 
   let concurrent_jobs = if matches.is_present("jobs") {
     if let Some(value) = matches.value_of("jobs") {
+      println!(
+          "{}",
+          crate::colors::yellow("Warning: --jobs flag with numeric value is deprecated. Use 'DENO_JOBS' environment variable with --jobs flag instead."),
+        );
```

Comment:
I'm surprised this passes the formatting check for `cargo fmt`. I guess because it's in a macro.